src/pages/AuthPage/AuthPage.style.jsx

```diff
@@ -0,0 +1,228 @@
+import styled from "styled-components";
+
+export const Auth = styled.article`
+    color: #858598;
```

Comment:
I think most colors will be reused on other places, so we could create some constants and use them in all places. 
FE:
```
file assets/themes/colors.js

const color1 = rgb(0,0,0);
const color2 = rgb(50,50,0);
export const ACCENT = color1;
export const INPUT_BORDER = color2;
export const ACTIVE_TEXT = color2;
```

And so on.